solutions/drmmessages/drmmessages.py

```diff
@@ -0,0 +1,40 @@
+"""
+https://open.kattis.com/problems/drmmessages
+
+Author: Sam Westigard
+"""
+from sys import stdin
+
+def rotationValue(string):
+  total = sum([ord(i)-65 for i in string])
```

Comment:
It would cleaner to me if we use `ord('A')` instead of hard-coding the value.
```suggestion
  total = sum([ord(i) - ord('A') for i in string])
```